index.html

```diff
@@ -1986,76 +1986,103 @@ <h1> Material beyond this point has not been assigned.  The descriptions below a
 </p>
 
 
-<h3>Forward Kinematics</h3>
+<h3>Forward Kinematics Files</h3>
 <p>
-Assuming proper completion of Just Starting Mode, you are now ready for implementation of robot forward kinematics.  Ensure the following files are included (within script tags) in your "home.html".  You will modify these files for implementing FK:
+Assuming proper completion of Just Starting Mode, you are now ready for implementation of robot forward kinematics.  The following files are included (within script tags) in "home.html".  You will modify these files for implementing FK:
 
 <ul>
-<li><p>"kineval/kineval_robot_init.js" for initializing your robot object based on a given description object; modification is require to add parent and child references to each link</p></li>
+<li><p>"kineval/kineval_robot_init_joints.js" for initializing your robot object based on a given description object; modification of this file is required to add parent and child references to each link</p></li>
 <li><p>"kineval/kineval_forward_kinematics.js" for implementing (a recursive) traversal over joints and links to compute transforms; traversal of forward kinematics is invoked from kineval.robotForwardKinematics() within my_animate() in home.html</p></li>
- <li><p>"kineval/kineval_matrix.js" for the implementation of your vector and matrix routines, such as for matrix multiplication, matrix generation, etc. in kineval_matrix.js</p></li>
+ <li><p>"kineval/kineval_matrix.js" for the implementation of your vector and matrix routines, such as for matrix multiplication, matrix generation, etc.</p></li>
 </ul>
 </p>
 
+<h3>Core Matrix Routines</h3>
 
-<h3> Robot Examples and Initialization</h3>
 <p>
-Each file in the "robots" subdirectory contains code to create a robot data object .  This data object is initialized the kinematic description of a robot (as well as some meta information and rendering geometries).  The kinematic description defines a hierarchical configuration of the robot's links and joints.  This description is a subset of the <a href="http://wiki.ros.org/urdf">Unified Robot Description Format (URDF)</a> converted into JSON format.  The basic features of URDF are described in <a href="http://wiki.ros.org/urdf/Tutorials/Create%20your%20own%20urdf%20file">this tutorial</a>.  
+A good place to start with your FK implemetation is writing and testing the core matrix routines. "kineval/kineval_matrix.js" contains function stencils for all required linear algebra routines. You will need to uncomment and fill in all the functions provided in this file <b>except matrix_pesudoinverse</b>. You do <b>not</b> need to implement the pseudoinverse calculation for this assignment; you should leave the matrix_pseudoinverse function commented out.
```

Comment:
I went into a little more detail about the matrix routines here and tried to make it very clear that they do not need to implement the pseudoinverse (yet).